src/Makefile

```diff
@@ -47,6 +47,7 @@ SRCS=\
      time.cc \
      base64.cc \
      format.cc \
+     health_reporter.cc \
```

Comment:
Why not call this `health_check.cc`?

Reply:
I do wish you'd done this rename as a separate commit. Can we rewrite the history?